spec/models/user_spec.rb

```diff
@@ -328,6 +326,14 @@
     end
   end
 
+  describe '#student_team' do
+     it 'return student current team' do
+      team    = FactoryGirl.create(:team)
```

Comment:
Afaik, it's not necessary to actually "create" all the records here. Building should be fine.